apps/communications/contacts/test/marionette/lib/views/contact-list/view.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/* global module */
+
+var ContactsListAccessor = require('./accessors');
+
+function ContactsListView(client) {
+	this.client = client;
+	this.accessors = new ContactsListAccessor(client);
+}
+
+ContactsListView.prototype = {
+	get firstContact() {
+		return this.accessors.firstContact;
+	},
+
+	get firstFavorite() {
+		return this.accessors.firstFavorite;
+	},
+
+	waitSlideLeftDetails: function() {
+		var element = this.client.findElement(
+		this.accessors.detailsSelector),
+        location;
+    	var test = function() {
```

Comment:
[Like there](https://github.com/mozilla-b2g/gaia/pull/33928/files#r50830027), we can be more explicit about the content of this `test`. How about: `waitUntilPanelReachedLeftBorder()` ?

If you think it's too redundant, we can inline it.